backend/src/apiserver/main.go

```diff
@@ -235,6 +241,14 @@ func startHttpProxy(resourceManager *resource.ResourceManager) {
 	// Register a handler for Prometheus to poll.
 	topMux.Handle("/metrics", promhttp.Handler())
 
+	pvValidateWebhook, err := webhook.NewPipelineVersionWebhook(controllerClient)
```

Comment:
The other endpoints shouldn't be running if in Kubernetes webhook only mode.

Reply:
Sure, I will make the changes accordingly.